backend/composer/models.py

```diff
@@ -239,6 +238,15 @@ class Meta:
         verbose_name_plural = "Anatomical Entities"
 
 
+class Layer(AnatomicalEntity):
+    ...
+
+
+class Region(AnatomicalEntity):
+    ...
+    associated_layer = models.ForeignKey(Layer, on_delete=models.CASCADE, related_name='regions')
```

Comment:
I had to make this a ForeignKey instead of a ManyToMany because when we create a connectivity statement with a region-layer pair we need to know exactly what is the region-layer pair and we expect that information to be under the umbrella of the anatomical entity model. In other words, if we want to add a complex origin to a connectivity statement we need to add an anatomical entity that contains information about the specific region and the specific layer.

Reply:
@zsinnema A region can have multiple layers yes, but only a pair region layer represents an anatomical entity. For example:

Region: Segment L1
Layers: Lamina 1, Lamina 2, Lamina 3, Lamina 4, etc...

The anatomical entities that we have, as far as I understand, are:
- Segment L1 + Lamina 1
- Segment L1 + Lamina 2
- ....


One relevant point to consider too is that in the Connectivity Statement model we refer to AnatomicalEntity (not RegionLayerPair).
    `origins = models.ManyToManyField(AnatomicalEntity, related_name='origins_relations')`